src/components/Mission/Mission.jsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+import styled from "styled-components";
+import { Typography, Box, makeStyles, Grid } from "@material-ui/core";
+
+import valuesIcon from "../../assets/images/value.png";
+import visionIcon from "../../assets/images/vision.png";
+import missionIcon from "../../assets/images/mission.png";
+
+import './styles.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import "./styles.css";
```